frontend/src/app/shared/components/send-alarm-group-interface/send-alarm-group-interface.component.html

```diff
@@ -0,0 +1,185 @@
+                <label class="flex-shrink-1 text-nowrap"
+                    >Anzahl Fahrzeuge:</label
+                >
+                <input
+                    [(ngModel)]="firstVehiclesCount"
+                    #firstVehiclesInput="ngModel"
+                    appIntegerValidator
+                    required
+                    min="0"
+                    step="1"
+                    name="firstVehiclesCount"
+                    type="number"
+                    tabindex="3"
+                    class="form-control flex-grow-1 flex-shrink-1 ms-2"
+                    style="width: 1px"
+                />
+            </div>
+            <app-display-validation [ngModelInput]="firstVehiclesInput" />
+            <div class="d-flex flex-row align-items-center mt-2">
+                <label class="flex-shrink-1 text-nowrap"
+                    >Die ersten {{ firstVehiclesCount }} Fahrzeuge fahren
+                    nach:</label
+                >
+                <button
+                    [ngbPopover]="selectFirstVehiclesTargetTemplate"
+                    #selectFirstVehiclesTargetPopover="ngbPopover"
+                    placement="bottom"
+                    [disabled]="
+                        (transferPoints?.length ?? 0) === 0 ||
+                        firstVehiclesCount < 1
+                    "
+                    tabindex="4"
+                    class="btn btn-outline-primary flex-grow-1 flex-shrink-1 ms-2 d-flex flex-row"
+                    style="width: 1px"
+                    type="button"
+                >
+                    <span
+                        class="flex-grow-1 flex-shrink-1 text-nowrap overflow-hidden"
+                        style="text-overflow: ellipsis; width: 1px"
+                    >
+                        {{
+                            selectedFirstVehiclesTarget
+                                ? selectedFirstVehiclesTarget.name
+                                : 'Ziel für die ersten Fahrzeuge auswählen'
+                        }}
+                    </span>
+                    <app-hotkey-indicator
+                        [hotkey]="selectFirstVehiclesTargetHotkey"
+                        class="flex-shrink-1 ms-1"
+                    />
+                    <i class="bi bi-caret-down-fill flex-shrink-1 ms-1"></i>
+                </button>
+                <ng-template #selectFirstVehiclesTargetTemplate>
+                    <app-searchable-dropdown
+                        *ngIf="(transferPoints?.length ?? 0) !== 0"
+                        [options]="transferPoints!"
+                        (selected)="
+                            selectedFirstVehiclesTarget = $event;
+                            selectFirstVehiclesTargetPopover.close()
+                        "
+                    />
+                </ng-template>
+            </div>
+            <span
+                *ngIf="firstVehiclesCount > 0"
```

Comment:
```suggestion
                *ngIf="firstVehiclesCount > 1"
```

I don't think any sensible exercise would have empty alarm groups. Also, you then don't have to fix the grammar here.

Reply:
Fixed in 75dbed9e0f636794cadb3d79edbcf011a66e3604